tools/text_processing_deployment/alignment.py

```diff
@@ -0,0 +1,165 @@
+import os
+import string
+from argparse import ArgumentParser
+from typing import List
+
+import pynini
+from pynini import Far
+
+from nemo.utils import logging
+
+
+"""
+This files takes 1. FST path 2. entire string 3. start position of substring 4. end (exclusive) position of substring
+and returns 1. mapped output string 2. start and end indices of mapped substring
+
+Usage: 
+
+python alignment.py <fst file> \"2615 Forest Av, 1 Aug 2016\" 22 26
```

Comment:
could you please list limitations, e.g.  expected behavior on cases that require order change